### PR TITLE
style changes for zipcode form, button and placement

### LINKED ISSUE #420: Zipcode Selector - CSS
Zip Code Selector styles seems were not working as they are supposed to be . We lost them due to some other changes does to the css 

image

### PR DESCRIPTION
**Pull request by kmurugulla**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #420 

Test URLs:
- Before: https://main--sling--aemsites.aem.live/programming/sports/pro-football
- After: https://420-zip--sling--aemsites.aem.live/programming/sports/pro-football

### CODE REVIEW

**Comment by rusmeenkhan1** [comment]
@kmurugulla there are a few small suggestions:
1) width of ours is bit less (110px) and theirs is 127px. 
2) padding between location icon and 0 is more in our case
3) Background color seems different in zipcode block (they are using  rgb(41, 41, 56) as background color ).

File: aemedge/blocks/zipcode/zipcode.css [hunk 1]
```diff
@@ -146,14 +147,21 @@ header nav[aria-expanded="false"] .zipcode-wrapper {
 
 .zip-update-btn {
     margin-bottom: 20px;
-    width: 100%;
+    width: 265px;
```
Comment (helms-charity) [inline aemedge/blocks/zipcode/zipcode.css:150]: this is too wide for mobile
image

**Comment by kmurugulla** [comment]
image

Tried to get the styles as close as possible for zipcode selection experience . Please review @rusmeenkhan1 and @helms-charity

**Review by helms-charity — COMMENTED** [review]
Still looking good

**Pull request merged**